### ISSUE TITLE
BE Support

### ISSUE THREAD

**Issue by FatalXFury** [cfogrady/DIM-Modifier#21]
Is there a way to make written BEMs work?

**Comment by cfogrady** [comment]
You're going to need to give me more to go on... That's literally what the 2.0 release from 2 days ago did... Support modifying dumps from BEMs.

**Comment by FatalXFury** [comment]
Sorry, should've been more specific. I used Dimtool to write a BE bin onto my BEM card. It gave me an error once i inserted it. How can i make it work? Would Mr.Blinky's Dimtool need to be updated for BEMemory support?

**Comment by cfogrady** [comment]
The image write and extraction hasn't changed. So My Blinky should work fine. Sounds like maybe a modification of a bad read? What error code did the BE give you?
Also make sure that the BE bin is being written back to the same BE Memory card you read from. The OTP prevents a BE Memory image from working if it is written to a BE Memory card with a different card id than it originally came from. My tool can't change that.

In other words. When you modify an image that came from a MHA vol 1 card, when you write it has to go back onto a MHA vol 1 card. Similarly a modified MHA vol 2 card image has to go back onto a MHA vol 2 physical card.

**Comment by FatalXFury** [comment]
> The image write and extraction hasn't changed. So My Blinky should work fine. Sounds like maybe a modification of a bad read? What error code did the BE give you? Also make sure that the BE bin is being written back to the same BE Memory card you read from. The OTP prevents a BE Memory image from working if it is written to a BE Memory card with a different card id than it originally came from. My tool can't change that.
> 
> In other words. When you modify an image that came from a MHA vol 1 card, when you write it has to go back onto a MHA vol 1 card. Similarly a modified MHA vol 2 card image has to go back onto a MHA vol 2 physical card.

That explains it. What i was trying to do was writing a bin file from a different BEM onto my 25th BEM. Is there no way to bypass this?

**Comment by cfogrady** [comment]
None. The OTP is write once. There are ways to do it for a card that has no OTP set yet, but you can't change one BE Memory into a different one.
